Documentation update for crystal collimators

MarcoDAndrea opened the pull request:
The "Special elements" section of the user manual has been updated with information on crystal collimators, including the description of the additional parameters required by the collimator database as well as appropriate references.

vkbo commented on doc/user_manual/chSpecialElements.tex:
```diff
@@ -2035,6 +2035,31 @@ \subsubsection{Additional Collimator Settings}
+
+This flag enables the collimator specified in \texttt{name} to be treated as a crystal collimator.
+Crystal collimators require an additional set of parameters described in the following:
+
```
can you add `\index{crystal collimator}` at the end of the first paragraph? It adds a reference in the index at the back of the manual.

vkbo commented on doc/user_manual/chSpecialElements.tex:
```diff
@@ -2035,6 +2035,31 @@ \subsubsection{Additional Collimator Settings}
 
+\paragraph{Crystal Collimators} \texttt{CRYSTAL name Bend XDim YDim Thick Tilt MisCut Orient}\\
+
+This flag enables the collimator specified in \texttt{name} to be treated as a crystal collimator.
```
Change "This flag enables the collimator specified"
To "This flag enables the collimator or collimator family specified"

amereghe commented on doc/user_manual/theBibliography.tex:
```diff
@@ -277,5 +277,13 @@
+    \bibitem{crystal:2}
+    D.~Mirarchi, ``Crystal collimation for LHC'',
+    Ph.~D.~thesis, Imperial College, London, UK (2015).
+    %
```
what about adding [this ref](http://cds.cern.ch/record/2694336?ln=en)?

Moreover, does the imported code take into account the [work by F. Forcher](http://cds.cern.ch/search?ln=en&as=1&m1=a&p1=forcher&f1=author&op1=a&m2=a&p2=&f2=title&op2=a&m3=a&p3=&f3=&action_search=Search&c=CERN+Document+Server&sf=&so=a&rm=&rg=100&sc=1&of=hb)?

MarcoDAndrea commented:
Thank you @vkbo and @amereghe for your comments. I implemented all your suggestions, added a couple more references as suggested by Daniele and removed the "to be implemented" note for crystals among the collimator families. I also added some comments in the crystal module to clarify the meaning of each process flag.
